ui/tests/e2e/extensions.spec.js

```diff
@@ -40,17 +40,14 @@ test.describe('Extensions Section Tests', () => {
     await expect(page.getByTestId('performance-analysis-heading')).toBeVisible();
     const performanceEnableButton = page.getByTestId('performance-analysis-enable-btn');
     await expect(performanceEnableButton).toBeVisible();
-    await expect(performanceEnableButton).toBeDisabled();
+    await expect(performanceEnableButton).toBeEnabled();
   });
 
-  test('Verify Kanvas Details', { tag: '@unstable' }, async ({ page, context }) => {
+  test('Verify Kanvas Details', { tag: '@unstable' }, async ({ page }) => {
```

Comment:
lets also remove these unstable tags as we are gaining confidence that they will pass now after all your efforts.